src/js_tests/.eslintrc

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
In my environment (Visual Studio Code), the ESLint plugin was complaining about the jasmine functions in the test files. So I created this config which is autodetected for the test folder then, and the live validation of the ESLint plugin works.

Not sure if you want to keep this, maybe your IDEs can be differently configured?

Reply:
We already have a `.eslintrc` file for tests, it is available at `src/.eslint-jasmine` and it is used by grunt. So maybe we should move it there instead of using your file.

Reply:
Yes, I wasn't sure if the `.eslintrc-jasmine` was used somewhere else, and didn't want to remove it directly. But I can adjust that if it is fine